api/db/seeds/data/pix-admin-roles-builder.js

```diff
@@ -1,6 +1,14 @@
-const { PIX_SUPER_ADMIN_ID } = require('./users-builder');
-const { ROLES } = require('../../../lib/domain/constants').PIX_ADMIN;
+const {
+  PIX_SUPER_ADMIN_ID,
+  PIX_SUPPORT_ID,
+  PIX_METIER_ID,
+  PIX_CERTIF_ID,
+} = require('./users-builder');
+const { ROLES } = require('../../../lib/domain/constants');
 
 module.exports = function pixAdminRolesBuilder({ databaseBuilder }) {
   databaseBuilder.factory.buildPixAdminRole({ userId: PIX_SUPER_ADMIN_ID, role: ROLES.SUPER_ADMIN });
+  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_SUPPORT_ID, role: ROLES.SUPPORT });
+  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_METIER_ID, role: ROLES.METIER });
+  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_CERTIF_ID, role: ROLES.CERTIF });
```

Comment:
Y'a une modif qui me semblait avoir été corrigée qui est passée à la trappe 🤔 le db:reset plante à cause de ça : 
```suggestion
const { PIX_ADMIN } = require('../../../lib/domain/constants');

module.exports = function pixAdminRolesBuilder({ databaseBuilder }) {
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_SUPER_ADMIN_ID, role: PIX_ADMIN.ROLES.SUPER_ADMIN });
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_SUPPORT_ID, role: PIX_ADMIN.ROLES.SUPPORT });
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_METIER_ID, role: PIX_ADMIN.ROLES.METIER });
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_CERTIF_ID, role: PIX_ADMIN.ROLES.CERTIF });
```

Reply:
j'irai plutot avec 
```suggestion
const { ROLES } = require('../../../lib/domain/constants').PIX_ADMIN;

module.exports = function pixAdminRolesBuilder({ databaseBuilder }) {
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_SUPER_ADMIN_ID, role: ROLES.SUPER_ADMIN });
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_SUPPORT_ID, role: ROLES.SUPPORT });
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_METIER_ID, role: ROLES.METIER });
  databaseBuilder.factory.buildPixAdminRole({ userId: PIX_CERTIF_ID, role: ROLES.CERTIF });
```

on a fait ca sur tout le reste